sdk/identity/identity/package.json

```diff
@@ -55,7 +55,7 @@
     "format": "dev-tool run vendored prettier --write --config ../../../.prettierrc.json --ignore-path ../../../.prettierignore \"src/**/*.ts\" \"test/**/*.ts\" \"samples-dev/**/*.ts\" \"*.{js,json}\"",
     "check-format": "dev-tool run vendored prettier --list-different --config ../../../.prettierrc.json --ignore-path ../../../.prettierignore \"src/**/*.ts\" \"test/**/*.ts\" \"samples-dev/**/*.ts\" \"*.{js,json}\"",
     "integration-test:browser": "echo skipped",
-    "integration-test:node": "dev-tool run test:node-js-input -- --timeout 180000 'dist-esm/test/public/node/*.spec.js' 'dist-esm/test/internal/node/*.spec.js'",
+    "integration-test:node": "dev-tool run test:node-js-input --no-esm=true -- --loader=../../../common/tools/esm4mocha.mjs --timeout 180000 'dist-esm/test/public/node/*.spec.js' 'dist-esm/test/internal/node/*.spec.js'",
```

Comment:
identity's integration tests use the recordings as well, for `<reasons>`, so we need the test proxy running. Easiest way is to keep using dev-tool commands

Reply:
Yes I was going to add an option then switch most of our packages over to use that. Eventually also would like to make using the loader default behavior. `--no-esm` doesn't sound like a great name but I haven't got a better one either.

Reply:
I was thinking maybe something like "loader" with a default of `esm`?

Then, you could omit it for existing behavior or pass `--loader=esm4mocha` for the new mocha loader. Maybe that would be nicer? I'll push that up shortly because I also don't love "no-esm" - let me know how that looks once I push the changes up!

Reply:
I feel that `--loader=` may be confusing, sounding like you could pass any strings there but they get ignored.  How about `--use-esm-workaround=false` with default value of `true`? I think there are only a handful packages that doesn't work immediately now with the `esm4mocha` loader.  Once I switch majority over, I can make the default value `false` then added ``--use-esm-workaround=true` to those a few packages.

Reply:
Yeah good point and I agree with `loader` being confusing... let me push up a change with your suggestion and we can see if that reads better 👍